src/ui/buttons.html

```diff
@@ -21,11 +21,16 @@
     <script src="js/extern/jquery-1.9.0.min.js"></script>
     <script>
         $(document).ready(function() {
-          $.getScript('js/Loader.js', function() {
-            Loader.loadScripts([ 'js/Buttons.js' ], function() {
-                Login.showLoginHeader(Buttons);
+            $.ajax({
+                url: 'js/Loader.js',
+                dataType: 'script',
+                cache: true,
+                success: function() {
+                    Loader.loadScripts([ 'js/Buttons.js' ], function() {
+                        Login.showLoginHeader(Buttons);
+                    });
+                }
```

Comment:
Just a note: all of these change the horizontal whitespace --- our current standard is 2 char indents for JS.